boutiques_descriptors/micapipe_0_2_2.json

```diff
@@ -0,0 +1,713 @@
+{
+  "name":           "micapipe",
+  "tool-version":   "0.2.2",
+  "container-image": {
+    "image": "micalab/micapipe:v0.2.2",
+    "index": "docker://",
+    "type": "singularity"
+  },
+  "schema-version": "0.5",
+  "author":         "Multimodal Imaging and Connectome Analysis (MICA) Lab ",
+  "url":            "https://github.com/MICA-MNI/micapipe",
+  "description":    "BIDS App for micapipe. Please cite: Cruces RR, Royer J, Herholz P, Lariviere S, Vos de Wael R, Paquola C, Benkarim O, Park BY, Degre-Pelletier J, Nelson MC, DeKraker J, Leppert IR, Tardif C, Poline JB, Concha L, Bernhardt BC. Micapipe: A pipeline for multimodal neuroimaging and connectome analysis. Neuroimage. 2022 Nov;263:119612. doi: 10.1016/j.neuroimage.2022.119612. Epub 2022 Sep 5. PMID: 36070839. doi: 10.1016/j.neuroimage.2022.119612",
```

Comment:
This tool is described as a 'BIDS App' but it doesn't seem to conform to the BIDS app specification. A BIDS App MUST follow a strict command-line conventions, e.g. `toolname bidsdataset outputdir analysislevel [--participant_label 1 2 3 4] [otherargs]`. The bidsdataet and outputdir arguments are positional and do NOT have flags on the command line. See the BIDS specification.

It is not clear to me what exactly this tool requires as input. Is it a BIDS dataset or a BIDS subject ? If it is a BIDs subject, then why are there an option `-sub subjectid`? If it is a BIDS dataset, then the subject name should be specified using `-participant_label`, according to the BIDS App specifications.

Reply:
Base on these comments, I would also make sure to rename everything properly. If the input is a BIDS dataset, then do not call it 'subject_dir', that's misleading.